test/e2e/specs/a11y.test.js

```diff
@@ -18,6 +23,23 @@ describe( 'a11y', () => {
 		await newPost();
 	} );
 
+	it( 'passes aXe JavaScript Accessibility API checks', async () => {
+		const handle = await page.evaluateHandle( `
```

Comment:
We've now got an [`axe-puppeteer`](https://github.com/dequelabs/axe-puppeteer) for this, and we hope to get it working today (at WordCamp).

Reply:
Awesome, as far as I can see it contains many useful helpers. That will make everything easier.